src/components/Input/Input.js

```diff
@@ -41,6 +42,7 @@ const Input = ({
           name={name}
           id={`${sluggified}__input`}
           disabled={disabled}
+          as={tag}
```

Comment:
Add to propTypes please. Do you want to include a defaultProp?

Reply:
Hm.. Skipped lint. I have added a default value of `input`. Thanks!

Reply:
Perfect